lib/websocket.js

```diff
@@ -91,8 +92,29 @@ var client = {
                 }
                 client._callbacks[data.Identifier] = null;
             }
-            else if (data.Identifier == 0) {
-                if (data.Message && !client._toolsCallbacks['ignore'](data.Message)) {
+
+            else if (data.Identifier == -1) {
```

Comment:
this is my fix for issue #3 
https://github.com/Kirbo/rust-server-cli-rcon/issues/3